### ISSUE TITLE
Human-friendly format not works

### ISSUE THREAD

**Issue by jnst** [lucagrulla/cw#24]
Thanks for 2.0.0 release 🎉

## Log not found

```bash
$ date -u
Wed Oct 17 03:32:52 UTC 2018
```
It was correctly outputted by the way used until. (FYI: I am in the time zone of JST so some logs below are displayed at +0900)

```bash
$ cw -p my-profile tail -ft /ecs/dev-my-server
2018-10-17T12:32:42 - [2018-10-17 03:32:42.533]    0 ms -- /health/check [:GET]   200 [] "-" "-" ::ffff:10.0.1.137
2018-10-17T12:32:49 - [2018-10-17 03:32:49.065]    0 ms -- /health/check [:GET]   200 [] "-" "-" ::ffff:10.0.0.174
```

But it is not output as a new way.

```bash
$ cw -p my-profile tail -ft /ecs/dev-my-server \* 5m
```

It was not found even if full log stream name was specified.

```bash
$ cw -p my-profile tail -ft /ecs/dev-my-server ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc 5m
No such log stream(s).

$ cw -p my-profile ls streams /ecs/dev-my-server | grep bcc9fc24
ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc
```

## panic: send on closed channel

About the case where wrong args is specified, I sometimes encounter panic error.

```
$ cw -p my-profile tail -ft /ecs/dev-my-server 2m
No such log stream(s).
panic: send on closed channel

goroutine 55 [running]:
github.com/lucagrulla/cw/cloudwatch.(*CW).Tail.func4(0xc000056680, 0xc000056501, 0x1012bd2)
	/Users/lucagrulla/Projects/cw/cloudwatch/tail.go:150 +0x175
github.com/aws/aws-sdk-go/service/cloudwatchlogs.(*CloudWatchLogs).FilterLogEventsPagesWithContext(0xc0001ae050, 0x1fb1128, 0xc0000240b0, 0xc0001000f0, 0xc000056580, 0x0, 0x0, 0x0, 0xc0001a8b01, 0xc0001000f0)
	/Users/lucagrulla/go/pkg/mod/github.com/aws/aws-sdk-go@v1.14.17/service/cloudwatchlogs/api.go:2288 +0x116
github.com/aws/aws-sdk-go/service/cloudwatchlogs.(*CloudWatchLogs).FilterLogEventsPages(0xc0001ae050, 0xc0001000f0, 0xc000056580, 0x0, 0x0)
	/Users/lucagrulla/go/pkg/mod/github.com/aws/aws-sdk-go@v1.14.17/service/cloudwatchlogs/api.go:2260 +0x90
github.com/lucagrulla/cw/cloudwatch.(*CW).Tail.func5(0xc0001cc2d0, 0xc00004d480, 0xc0001a8b40, 0xc0001c45d8, 0x0, 0xc00004d450, 0xc000024ad7, 0xc0001ae058, 0xc000056580)
	/Users/lucagrulla/Projects/cw/cloudwatch/tail.go:175 +0x235
created by github.com/lucagrulla/cw/cloudwatch.(*CW).Tail
	/Users/lucagrulla/Projects/cw/cloudwatch/tail.go:171 +0x40d
```

**Comment by jnst** [comment]
Other case.

```bash
$ cw -p my-profile tail -ft /ecs/dev-my-server \* 2h:30m
2018/10/17 14:03:41 1 validation error detected: Value '-62135596800000' at 'startTime' failed to satisfy constraint: Member must have value greater than or equal to 0
```

**Comment by lucagrulla** [comment]
few things:
- sometimes in your exmaples you forget to pass the stream name. That is causing you troubles
- sometimes in your examples you use the wrong syntax: it's 2h30m without column, 2h:30minutes is not accepted

I'm also confused about the pamis msg you see...there should not be any more explicit panic calls in the call. Silly question but: are you sure your update worked well?

cw --version => 2.0.0 ?

**Comment by jnst** [comment]
## Version

I already updated to latest.

```bash
$ cw --version
2.0.0
```

Btw, I haven't installed it via `go get`.

```bash
$ echo {$GOBIN}
/Users/jnst/go/bin

$ ls {$GOBIN} | grep cw
```

And I tried re-install.

```bash
$ brew uninstall -force cw
$ brew install cw

🍺  /usr/local/Cellar/cw/2.0.0: 7 files, 9.8MB, built in 2 seconds
```

But the result is the same and a panic error occurs.

## About pass the stream name

>sometimes in your exmaples you forget to pass the stream name. That is causing you troubles

In my understanding the following format is correct but it does not work well.

```bash
$ cw -p my-profile tail -ft /ecs/dev-my-server \* 1m

$ cw -p my-profile tail -ft /ecs/dev-my-server ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc 1m
No such log stream(s).
```

## 2h:30m syntax

>sometimes in your examples you use the wrong syntax: it's 2h30m without column, 2h:30minutes is not accepted

Ah OK, this case is document issue.  Please fix `README.md`.

```
cw tail -f my-log-group my-log-stream-prefix 2h:30m to start from 2 hours and 30 minutes ago.
```

>I'm also confused about the pamis msg you see...there should not be any more explicit panic calls in the call.

In my environment almost always a panic error occurs. 🤔

```bash
$ cw -p my-profile tail /ecs/dev-my-server 1m
No such log stream(s).
panic: send on closed channel

goroutine 32 [running]:
github.com/lucagrulla/cw/cloudwatch.(*CW).Tail.func4(0xc0001b8440, 0xc0001b8301, 0x1012bd2)
	/Users/lucagrulla/Projects/cw/cloudwatch/tail.go:150 +0x175
github.com/aws/aws-sdk-go/service/cloudwatchlogs.(*CloudWatchLogs).FilterLogEventsPagesWithContext(0xc0001a8050, 0x1ea10f8, 0xc0000240b0, 0xc00049e0f0, 0xc0001b8380, 0x0, 0x0, 0x0, 0xc0001a2b01, 0xc00049e0f0)
	/Users/lucagrulla/go/pkg/mod/github.com/aws/aws-sdk-go@v1.14.17/service/cloudwatchlogs/api.go:2288 +0x116
github.com/aws/aws-sdk-go/service/cloudwatchlogs.(*CloudWatchLogs).FilterLogEventsPages(0xc0001a8050, 0xc00049e0f0, 0xc0001b8380, 0x0, 0x101275f)
	/Users/lucagrulla/go/pkg/mod/github.com/aws/aws-sdk-go@v1.14.17/service/cloudwatchlogs/api.go:2260 +0x90
github.com/lucagrulla/cw/cloudwatch.(*CW).Tail.func5(0xc0001d02d0, 0xc00004d490, 0xc0001a2b10, 0xc0001ce5d8, 0x0, 0xc00004d460, 0xc000024ad7, 0xc0001a8058, 0xc0001b8380)
	/Users/lucagrulla/Projects/cw/cloudwatch/tail.go:175 +0x235
created by github.com/lucagrulla/cw/cloudwatch.(*CW).Tail
	/Users/lucagrulla/Projects/cw/cloudwatch/tail.go:171 +0x40d
```

**Comment by lucagrulla** [comment]
This cannot work:
```bash
cw -p my-profile tail -ft /ecs/dev-my-server ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc 1m
```
because you are not passing either the stream name or \*.
ecs/dev-my-server ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc is the stream group.

**Comment by jnst** [comment]
### Step1

`/ecs/dev-my-server` is log group name.

```bash
$ cw -p my-profile ls groups | grep '/ecs/dev'
/ecs/dev-my-server
```

### Step2

`ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc` is log stream name.

```bash
$ cw -p my-profile ls streams /ecs/dev-my-server | grep 'bcc9fc24'
ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc
```

### Step3

I specified `log group name` and `log stream name`.  
It works.

```bash
$ cw -p my-profile tail -ft /ecs/dev-my-server ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc
2018-10-17T18:17:55 - [2018-10-17 09:17:55.113]    0 ms -- /health/check [:GET]   200 [] "-" "-" ::ffff:10.0.1.137
```

### Step4

But `5m` not works.

```bash
$ cw -p my-profile tail -ft /ecs/dev-my-server ecs/my-server/bcc9fc24-7184-4064-aad7-921dd0bf32fc 5m
No such log stream(s).
```

### Step5

also `*` not works.

```bash
$ cw -p my-profile tail -ft /ecs/dev-my-server * 5m
cw: error: unexpected dist, try --help
```

**Comment by lucagrulla** [comment]
I found the bug. It's not related to human friendy, it's related to specific timestamp.

I'll fix it as soon as I have some time.

**Comment by lucagrulla** [comment]
I think the specific bug has been fixed in 2.0.1
It's a bit complicated because it's all about Local and it's confusing tot test.

A potential extra feature is to allow to define Local as a valid location instead of UTC...but more work is needed to allow this.

 I'm going to wait for feedback before marked as close :-)

**Comment by jnst** [comment]
All the problems that I had confirmed were resolved 👍
